apps/backend/src/resolvers/mutations/AssignProposalsToInstrumentsMutation.ts

```diff
@@ -67,4 +67,51 @@ export class AssignProposalsToInstrumentsMutation {
       args
     );
   }
+
+  @Mutation(() => Boolean)
+  async assignXpressProposalsToInstruments(
+    @Args() args: AssignProposalsToInstrumentsArgs,
+    @Ctx() context: ResolverContext
+  ) {
+    const techniquesWithProposal =
+      await context.queries.technique.getTechniquesByProposalPk(
+        context.user,
+        args.proposalPks[0]
+      );
+
+    if (!techniquesWithProposal || techniquesWithProposal.length < 1) {
+      return false;
```

Comment:
I think it would be better to return a Rejection instead of false so that it's clearer when validation has failed and we can provide some context as to why.